lib/ui/src/app.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import ResizeDetector from 'react-resize-detector';
+import { inject, observer } from 'mobx-react';
+
+import { Mobile } from './components/layout/mobile';
+import { Desktop } from './components/layout/desktop';
+import Nav from './containers/nav';
+import Preview from './containers/preview';
+import Panel from './containers/panel';
+
+const Reset = styled.div(({ theme }) => ({
+  fontFamily: theme.mainTextFace,
```

Comment:
These css properties cascade down, meaning less stuff to do to get theming mostly right 99% of the places.